<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1713</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>linux200命令按钮</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>日志</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="textBrowser"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2"/>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>抖音roomid</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edtCommand">
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Mode">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>80</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>主题</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_DOUYIN">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>80</height>
          </size>
         </property>
         <property name="text">
          <string>抖音爬虫</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_BILIBILI">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>80</height>
          </size>
         </property>
         <property name="text">
          <string>B站爬虫</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
